foghorn-edge/templates/cleanup-sa-cluster-role-binding.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ClusterRoleBinding
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.